week-1.md

```diff
@@ -0,0 +1,83 @@
+Hi!
+
+Thanks heaps for signing up to help review the schnorr and taproot BIP drafts! We've had 163 sign ups, which is a great number (it's prime if nothing else!) and looking through the names there's a lot of experience there!
+
+> "Some people worry about developers attempting to take control, but the defense against that is for other developers to review their code and sound an alarm if they see something inappropriate." -- David Harding, [2017-07-28](https://dtrt.org/posts/minimum-majority-measure/)
+
+This first week's focus is on two things: getting setup to review the BIPs and taking a first pass at the taproot BIPs; see the [first section in the Curriculum cheat sheet](https://github.com/ajtowns/taproot-review/blob/master/Curriculum.md#intro--taproot).
+
+## Intro
+
+As a general reminder, BIPs are just proposals -- they're the first step on the path to making a change to Bitcoin, but taking that first step doesn't mean you won't immediately turn around and walk straight back to where you came from. If you look through the bip repository on github, you can see many BIPs that haven't progressed, some that have and have been walked back from, and others that have achieved consensus and implementation and are in active use today. If you're not familiar with the process, it can be instructive to have a look through how things have gone in the past:
+
+ * [BIPs repo](https://github.com/bitcoin/bips/)
+ * [BIP 2 -- BIP policy](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki)
+ * [BIP PRs](https://github.com/bitcoin/bips/pulls?utf8=%E2%9C%93&q=is%3Apr)
+
+ * Segwit address format PRs:
```

Comment:
I'm guessing item (with the two sub-items) is meant as an example of a case where something was proposed, worked on a bit, withdrawn in the face of reasoned opposition, re-submitted as a significantly different proposal (accomplishing the same goals), and then accepted.  If that's the case, I think this could either use some additional introduction to separate it from the previous list items (BIPs repo, BIP2, BIP PRs) or we could provide other examples, e.g. BIP140 normalized txid vs BIP141 segwit (technically BIP134 flexible transactions too, but lol) or BIP12 OP_EVAL vs BIP16 P2SH vs BIP17/18 luke-jr proposals.  If you want, I can probably dig up links showing the evolution of those proposals.

Reply:
some more words added. don't want to overload people with too much history -- just needs to be a springboard to get them going, not a comprehensive reference, so haven't added other examples. might be good to have an link to why bech32 was needed or some of its design docs?

Reply:
Current text looks good.  I think BIP173's existing Rationale section is pretty good and you already link that, so I think it's ok unless you wanted to draw more attention to it: https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki#Rationale , but if you want more, my favorite is this post by Maxwell: https://bitcointalk.org/index.php?topic=2624630.msg26762627#msg26762627

Reply:
added a link to the entire thread as an example of discussion, seems good